### PR TITLE
cmd/puppeth: integrate istanbul into puppeth

### PR DESCRIPTION
**Pull request by rjl493456442**

### CODE REVIEW

File: cmd/puppeth/wizard_genesis.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ func (w *wizard) makeGenesis() {
 			ByzantiumBlock:      big.NewInt(0),
 			ConstantinopleBlock: big.NewInt(0),
 			PetersburgBlock:     big.NewInt(0),
+			IstanbulBlock:       big.NewInt(0),
```
Comment (karalabe) [inline cmd/puppeth/wizard_genesis.go]: We can't enable this until Istanbul is finalized.

Comment (rjl493456442) [reply cmd/puppeth/wizard_genesis.go]: Should I change it? Or take care of this PR when Istanbul is finalized?

Comment (karalabe) [reply cmd/puppeth/wizard_genesis.go]: No, it's fine to finish this PR when Istanbul is fully defined

**Comment by karalabe** [comment]
We should probably pick this PR up again, finalize it and ship it in 1.9.5.

**Comment by holiman** [comment]
Is there an upstream-example over at the parity repo to crosscheck against? They usually publish examples

**Comment by holiman** [comment]
Is parity istanbul definition done now?

**Comment by rjl493456442** [comment]
@holiman Yes. But still need the review from Frank since he also works on the config for multi-clients in the past.

**Pull request merged**